packages/@webex/plugin-cc/src/AgentConfigService/AgentConfigService.ts

```diff
@@ -0,0 +1,140 @@
+import {WebexSDK, HTTP_METHODS} from '../types';
+import {
+  DesktopProfileResponse,
+  ListAuxCodesResponse,
+  ListTeamsResponse,
+  AgentResponse,
+} from './types';
+import HttpRequest from '../HttpRequest';
+
+export default class AgentConfigService {
+  agentId: string;
+  orgId: string;
+  webex: WebexSDK;
+  wccAPIURL: string;
+  requestInstance: HttpRequest;
+
+  constructor(agentId: string, orgId: string, webex: WebexSDK, wccAPIURL: string) {
+    this.agentId = agentId;
+    this.orgId = orgId;
+    this.webex = webex;
+    this.wccAPIURL = wccAPIURL;
+    this.requestInstance = new HttpRequest(this.webex);
+  }
+
+  /**
+   * Method to get Agent using CI.
+   * @returns {Promise<AgentResponse>} A promise that eventually resolves to an API response.
+   */
+
+  public async getUserUsingCI(): Promise<AgentResponse> {
+    try {
+      const URL = `${this.wccAPIURL}organization/${this.orgId}/user/by-ci-user-id/${this.agentId}`;
```

Comment:
move the path to Constants or at a central place

organization/${this.orgId}/user/by-ci-user-id

Reply:
I implemented it before but when I have a call with Priya, she mentioned to remove this constants.